### PR TITLE
Update chairs and teams for Contributor Experience

### PR DESCRIPTION
**Pull request by cblecker**
ref #4415

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -1070,9 +1070,6 @@ sigs:
     - github: Phillels
       name: Elsie Phillips
       company: Red Hat
-    - github: parispittman
```
Comment (nikhita) [inline sigs.yaml]: Let's also add Bob here and wait until EOD to merge for lazy consensus?

**Comment by nikhita** [comment]
For the record: https://groups.google.com/forum/#!topic/kubernetes-sig-contribex/ch8qph663D8

**Comment by nikhita** [comment]
/lgtm

/hold
until Monday January 20th, at 2pm PT (5pm ET)

**Comment by cblecker** [comment]
/hold cancel
cc: @mrbobbytables

**Pull request merged**